packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -125,12 +122,16 @@ class FrameFlameChart extends CoreElement {
     _frame = frame;
     _resetChart();
 
-    if (_debugEventTrace && frame != null) {
-      final StringBuffer buf = new StringBuffer();
+    if (debugTimeline && frame != null) {
```

Comment:
Nit: you could implement Diagnosticable for this sort of large debugging string output.